scripts/uberenv/uberenv.py

```diff
@@ -319,7 +319,7 @@ def setup_paths_and_dirs(self):
     def clone_repo(self):
         if not os.path.isdir(self.dest_vcpkg):
             # compose clone command for the dest path, vcpkg url and branch
-            vcpkg_branch = self.project_opts.get("vcpkg_branch", "master")
+            vcpkg_branch = self.project_opts.get("vcpkg_branch", "main")
```

Comment:
vcpkg has not changed their branch from master.

Reply:
Good catch. I'll fix that.

Reply:
Changed back.